corehq/apps/domain/forms.py

```diff
@@ -956,6 +958,19 @@ class DomainInternalForm(forms.Form, SubAreaMixin):
         required=False,
         min_value=1,
     )
+    use_custom_gsheet_limit = forms.ChoiceField(
+        label=ugettext_lazy("Set custom Google Sheet limit? Default is 20"),
```

Comment:
i think this needs to be updated to `gettext_lazy` due to the Django 3.0 changes?